js/app.js

```diff
@@ -1 +1,260 @@
+
+console.log(isThisMyName('John'));
+
+// 3
+function isThisBoolean(value) {
+    return value === false || value === true;
+}
+
+console.log(isThisBoolean(true));
+console.log(isThisBoolean(false));
+console.log(isThisBoolean('true'));
+
+// 4
+function getCircleArea(radius){
+    return Math.PI * radius * radius;
+}
+
+const radius = 5;
+const area = getCircleArea(radius);
+console.log(area); // 78.53981633974483
+
+// 5
+function getCirclesAreaSum(radius1, radius2) {
+    const area1 = getCircleArea(radius1);
+    const area2 = getCircleArea(radius2);
+
+
+    return area1 + area2;
+}
+
+const circlesAreaSum = getCirclesAreaSum(5, 10);
+console.log(circlesAreaSum); // 392.69908169872417
+
+// 6
+const quarterOf = (month) => {
+    return Math.ceil(month / 3);
+}
+
+// 7
+function convertMinutesToHours(minutes) {
+    return minutes / 60
+}
+
+const hours = convertMinutesToHours(75);
+console.log(hours); // 1.25
+
+// 8
+function isTextUppercase(text) {
+    return text === text.toUpperCase();
+}
+
+console.log (isTextUppercase('Hello')); // false
+console.log (isTextUppercase('HELLO')); // true
+
+// 9
+function isDivisible(n, x, y) {
+    return n % x === 0 && n % y === 0;
+}
+
+// 10
+function getBillboardPrice(text, costPerCharacter) {
+    return text.length * costPerCharacter
+}
+
+console.log(getBillboardPrice('Hello world!', 10));  // 120, since there are 12 characters
+console.log(getBillboardPrice('Hello world!', 15));  // 180
+console.log(getBillboardPrice('To be, or not to be', 20));  // 380, since there are 19 characters
+
+// 11
+function opposite(number) {
+    return (number * -1)
+}
+
+// 12
+function makeNegative(num) {
+    if (num >0) {
+        return (num * -1)
+    }
+}
+
+// 13
+function removeChar(str){
+    return str.slice(1, -1);
+};
+
+// 14
+function basicOp(operation, value1, value2){
+    if (operation === '+') {
+        return value1 + value2;
+    } else if (operation === '-') {
+        return value1 - value2;
+    } else if (operation === '*') {
+        return value1 * value2;
+    } else if (operation === '/') {
+        return value1 / value2;
+    } else {
+        return 0;
+    }
+}
+
+// 15
+function century(year) {
+    const century = Math.ceil (year / 100);
+    return century;
+}
+
+// 16
+function evenOrOdd(number) {
+    if (number % 2 === 0) {
+        return 'Even';
+    } else {
+        return 'Odd';
+    }
+}
+
+// 17
+function areYouPlayingBanjo(name) {
+    const firstLetterIsR = name.charAt(0).toLowerCase() === 'r';
+    if (firstLetterIsR) {
+        return name + " plays banjo";
+    } else {
+        return name + " does not play banjo";
+    }
+}
+
+// 18
+function rockPaperScissors(player1, player2) {
+    if (player1 === player2) {
+        return 0;
+    }
+
+    if (
+        (player1 === 'rock' && player2 === 'scissors') ||
+        (player1 === 'scissors' && player2 === 'paper') ||
+        (player1 === 'paper' && player2 === 'rock')
+    ) {
+        return 1;
+    } else {
```

Comment:
do you remember what to do when there is the else statement with the return statement?